### PR TITLE
Link to related codelab from Kubernetes page

### PR DESCRIPTION
**Pull request by rcarun**

### CODE REVIEW

File: setup/providers/kubernetes.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ hosted solutions:
 
 * [Google Container Engine](https://cloud.google.com/container-engine/)
 * [Azure Container
-  Service](https://docs.microsoft.com/en-us/azure/container-service/container-service-kubernetes-walkthrough)
+  Service](https://docs.microsoft.com/en-us/azure/container-service/container-service-kubernetes-walkthrough) [(codelab)](http://aka.ms/azspinlab)
```
Comment (ejizba) [inline setup/providers/kubernetes.md:39]: I don't think linking to the codelab here makes sense. This page assumes you have an existing Kubernetes cluster and you are setting up Spinnaker separately. The sentence preceding this even implies you just want to create a cluster by itself

That being said, the existing link is a pretty heavy doc page and we might be able to find a better link

**Comment by lwander** [comment]
This feels out-of-place since the links here are for setting up k8s alone in the context of getting spinnaker up and running. Injecting a codelab that does most of the basic spinnaker config for you without Halyard in the middle of it doesn't seem correct

File: setup/providers/kubernetes.md (hunk 1 above)
Comment (lwander) [reply setup/providers/kubernetes.md:39]: jinx

**Comment by ttomsu** [comment]
What's the word on this PR?

**Comment by lwander** [comment]
Closing for now because the PR is stale and link seems to be in the wrong place. Feel free to re-open or just create another PR if you want to link to the codelab elsewhere.

**Pull request closed without merging**